usaspending_api/common/exceptions.py

```diff
@@ -2,6 +2,13 @@
 from rest_framework import status
 
 
+class NoDataFoundException(APIException):
```

Comment:
TIL 204 status code. Would this apply to all our data points that yield no results, or just when the complete body of the json response is empty?

Reply:
Perhaps in the future we can use 204 more. I would lean toward that idea of an empty response body and 204 code